### PR TITLE
Add test for interpolating markdown strings

### PR DESCRIPTION
**Pull request by jlewis91**
@MichaelHatherly Looks like you were far faster than I at building out this new feature. I've been using this branch in a personal project and couldn't find a glitch so far. :) Reviewed the tests you have written, doesn't look like there are any obvious edge cases neglected as long as we keep away from the 'dollar math debacle', with the exception of the question: if an interpolated variable returns a string, do we then parse it for potential markdown syntax? Personally don't have a strong opinion, but I've encoded the status quo in a test within this PR.

### CODE REVIEW

**Comment by MichaelHatherly** [comment]
I think the most sensible approach would be to not do any kind of nested parsing to markdown for interpolated strings. If the user wants to interpolate markdown into their markdown they can pass the inner markdown themselves.

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "CommonMark"
 uuid = "a80b9123-70ca-4bc0-993e-6e3bcb318db6"
 authors = ["Michael Hatherly"]
-version = "0.7.2"
+version = "0.7.3"
```
Comment (MichaelHatherly) [inline Project.toml:4]: hmm, let me rebase `mh/interp` and see if this goes away.

**Comment by MichaelHatherly** [comment]
If you can rebase onto current `mh/interp` then I'll get this merged in there, merge it into master and just get a tagged release done. Thanks for the tests.

**Comment by jlewis91** [comment]
@MichaelHatherly Looks good now?

**Comment by MichaelHatherly** [comment]
thanks @jlewis91
